test/metabase/test_runner/assert_exprs/approximately_equal_js.clj

```diff
@@ -0,0 +1,15 @@
+(ns metabase.test-runner.assert-exprs.approximately-equal-js)
+
+;; this is only supposed to be a thing when doing macroexpansion FOR ClojureScript, and `cljs.test` isn't available in
+;; regular Clojure mode. For some reason I couldn't get Macrovich to do what I wanted here
+;;
+;; this is done in a preposterous way so that Cloverage doesn't trip up trying to compile it
```

Comment:
I think this might Just Work referencing `clojure.test` - that'll get remapped magically to `cljs.test` in CLJS mode when the macro's output contains `clojure.test/assert-expr`.

This can go in for now but there's probably a better way out there to be found.

Reply:
Yeah, but then we'd potentially be stomping on the Hawk version if pure-Clojure mode, since Hawk also implements this method for `clojure.test`. I wanted to try to watch out for that